### PR TITLE
#175494145 Created a bookNow form

### PR DESCRIPTION
**Pull request by Floride125-art**
## What does this PR do?
This PR indicates a BookNow pages that I have designed.
## Description of Task to be completed?
This book now page is a form that allow users to book accommodations, transport, or halls depending on the which places visitors wants to visit.
## How should this be manually tested?
-Clone it ```https://github.com/Soma-Technologies-Inc/visitAfrica-frontend.git```
-Run ```flutter pub get``` to install dependencies
-Run it by using  ```flutter run``` command in any text editor you wants
## Any background context you want to provide?
N/A
## What are the relevant pivotal tracker stories?

[175494145](https://www.pivotaltracker.com/n/projects/2474529/stories/175494145)
## Screenshots (if appropriate)
firstScreen
second Screen
thirdScreer
fourthScreen

## Questions:
N/A

### CODE REVIEW

File: lib/views/Bookhall.view.dart [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  Mybookdata booknow = Mybookdata(
+    "",
+    "",
+    "",
+    "",
+    "",
+    "",
+    "",
+  );
+  List<Mybookdata> mybookinfo = [];
```
Comment (VedaDom) [inline lib/views/Bookhall.view.dart]: Can you try to change the way you name your class and variable names?

File: lib/views/Bookhall.view.dart [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                        hall.text,
+                        time.text,
+                        timeTo.text,
+                        hourfrom.text,
+                        hourto.text,
+                      ));
+
+                      time.clear();
+                      phone.clear();
+                      hall.clear();
```
Comment (VedaDom) [inline lib/views/Bookhall.view.dart]: I this necessary?

File: lib/views/bookhotel.view.dart [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+  TextEditingController phone = new TextEditingController();
+  TextEditingController people = new TextEditingController();
+  TextEditingController hall = new TextEditingController();
+  TextEditingController time = new TextEditingController();
+  TextEditingController timeTo = new TextEditingController();
+  TextEditingController hourfrom = new TextEditingController();
+  TextEditingController hourto = new TextEditingController();
+  Mybookdata booknow = Mybookdata("", "", "", "", "", "", "");
+  List<Mybookdata> mybookdata = [];
```
Comment (VedaDom) [inline lib/views/bookhotel.view.dart]: These names should be self-explanatory.

File: lib/views/bookhotel.view.dart [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+                        time.text,
+                        timeTo.text,
+                        hourfrom.text,
+                        hourto.text,
+                      ));
+                      phone.clear();
+                      people.clear();
+                      hall.clear();
+                      time.clear();
+                      timeTo.clear();
```
Comment (VedaDom) [inline lib/views/bookhotel.view.dart]: Is that necessary?

**Review by VedaDom — CHANGES_REQUESTED** [review]

File: lib/views/bookhotel.view.dart (hunk 4 above)
Comment (Floride125-art) [reply lib/views/bookhotel.view.dart]: yes because I need to clear the information entered by the user on previous form

**Review by mujohn26 — APPROVED** [review]
Good!

**Pull request merged**